modules/python/python-dest.c

```diff
@@ -433,8 +441,8 @@ python_dd_init(LogPipe *d)
   gstate = PyGILState_Ensure();
 
   _py_perform_imports(self->loaders);
-  if (!_py_init_bindings(self) ||
-      !_py_init_object(self))
+  if (!_py_init_bindings(self)||
```

Comment:
This two line has only whitespace changes, it should not have.

Reply:
done.